demo/README.md

```diff
@@ -72,3 +135,46 @@ Use following functions to invoke a transaction or query.  Please note that any
 http://localhost:3000/api/cc/invoke
 http://localhost:3000/api/cc/query
 ```
+
+### 4. Frontend UI
+
+1. Build the Frontend UI docker image
+```
+cd $FPC_PATH/demo/client/frontend
+make build
+```
+
+2. Run the docker image
+```
+export NETWORK_NAME=fabricfpc_basic
+docker run --network $NETWORK_NAME -p 5000:5000 -d --name auction_frontend auction_frontend
```

Comment:
See above comment for backend regarding use of docker-compose instead of docker ...

Reply:
Similar to above, docker-compose can be used to build and run the auction frontend in one step. However the COMPOSE_PROJECT_NAME must be provided now.